lightbulb/ext/tasks/triggers.py

```diff
@@ -65,8 +68,25 @@ class CronTrigger(Trigger):
         """
         A crontab-based task trigger. Tasks will be run dependent on the given crontab.
 
+        .. note::
+            Keyword arguments are only used if no crontab expression is passed.
+
         Args:
-            crontab (:obj:`str`): Schedule that task executions will follow.
+            crontab (:obj:`str` | :obj:`None`): Schedule that task executions will follow.
+
+        Keyword Args:
+            month (:obj:`int` | :obj:`str` | :obj:`None`):
```

Comment:
Use below in docstring, current format is inconsistent with the rest of the library.
```py
Optional[Union[:obj:`int`, :obj:`str`]]
```

Reply:
Same for all other params